doc/source/universe.rst

```diff
@@ -126,6 +126,25 @@ For example, to construct a universe with 6 atoms in 2 residues:
 `See this notebook tutorial for more information. <examples/constructing_universe.ipynb>`_
 
 
+Constructing from SMILES
+------------------------
+
+Thanks to the interoperability with RDKit, a Universe can be constructed from a SMILES string with :meth:`Universe.from_smiles<MDAnalysis.core.universe.Universe.from_smiles>`. Since the SMILES string usually contains information about heavy atoms only, this method automatically adds the appropriate number of hydrogens on all atoms, by setting :code:`addHs=True`. It is also possible to generate coordinates with :code:`generate_coordinates=True`. This will allow RDKit to generate one or more conformers for the given molecule. For example, setting :code:`numConfs=10` will generate 10 conformers, and each conformer will be read as a frame by the Universe.
+
+Internally, calling the :meth:`Universe.from_smiles<MDAnalysis.core.universe.Universe.from_smiles>` method does the following:
+
+.. ipython:: python
+
+    from rdkit import Chem
+    from rdkit.Chem import AllChem
+
+    mol = Chem.MolFromSmiles("CCO") # ethanol
+    mol = Chem.AddHs(mol)
+    confids = AllChem.EmbedMultipleConfs(mol, numConfs=1) # 1 conformer
+    u = mda.Universe(mol, format="RDKIT")
+    u
```

Comment:
Could you please show an example of actually calling `Universe.from_smiles` with the arguments that you mention? Otherwise looks good, thanks!